Source/UI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -64,13 +84,15 @@ final class BillingFormViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupNavigationBar()
+
         view.backgroundColor = viewModel.style.mainBackground
         setupViewsInOrder()
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        navigationController?.setNavigationBarHidden(true, animated: false)
+        navigationController?.setNavigationBarHidden(false, animated: false)
```

Comment:
I think we never hide the navigation bar anymore in our SDK, so we can probably drop all references to `setNavigationBarHidden`. There's another place this is used.

```suggestion
```